packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -320,4 +377,19 @@ export class HeapDumpService {
   ) {
     return varContainer.name === extentValueEntry.keyDisplayValue;
   }
+
+  public isTriggerExtent(outerExtent: HeapDumpExtents) {
+    if (
+      (outerExtent.typeName === 'Boolean' ||
```

Comment:
We control the output of the trigger variables so we are certain that we can use `Boolean`, `List<`, etc and not have to do a case-insensitive comparison, right? Or do we also need to do case-insensitive comparisons?

Reply:
That's a good question. The answer is that we do not need a case insensitive comparison. While Apex, itself, may see Boolean and boolean in code as the same thing, the type is still Boolean. That being said, when outputting the types in the extent there is no variance in casing, it uses the actual type. These are the types for the Trigger context variables directly from the heapdump extents: 
"typeName": "List<Account>"
"typeName": "Map<Id,Account>"
"typeName": "Boolean"